storage/redis/peer_store_test.go

```diff
@@ -0,0 +1,54 @@
+package redis
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/alicebob/miniredis"
+
+	s "github.com/chihaya/chihaya/storage"
+)
+
+func createNew() s.PeerStore {
+	rs, err := miniredis.Run()
+	if err != nil {
+		panic(err)
+	}
+	redisURL := fmt.Sprintf("redis://@%s/0", rs.Addr())
+	ps, err := New(Config{GarbageCollectionInterval: 10 * time.Minute, PrometheusReportingInterval: 10 * time.Minute, PeerLifetime: 30 * time.Minute, RedisBroker: redisURL})
```

Comment:
Please add the other config fields to this, it is printing warning messages during the tests.

Reply:
done